Sage/carbon
# fix(menu): focus search using arrow keys - FE-4038

### Proposed behaviour

When a `Search` component is part of a `MenuItem` it is focusable by using the arrow keys.

### Current behaviour

When a `Search` component is part of a `MenuItem` it is currently not possible to focus by using the arrow keys.

### Checklist



### Additional context
N/A

### Testing instructions
- Search component should be focusable using arrow keys. 
design-system --> submenu-with-search

- Tab and Shift/Tab should respect the usual tabbing order in a web browser too.
